src/sentry/static/sentry/app/components/switch.jsx

```diff
@@ -17,26 +17,60 @@ class Switch extends React.Component {
   };
 
   render() {
-    let {size, isActive, isLoading, isDisabled, toggle, id} = this.props;
-    let switchClasses = classNames('switch', {
-      [`switch-${size}`]: size,
-      'switch-on': isActive,
-      'switch-changing': isLoading,
-      'switch-disabled': isDisabled,
-    });
+    let {size, isActive, isLoading, isDisabled, toggle, id, className} = this.props;
 
     return (
-      <div
+      <SwitchContainer
```

Comment:
yeah I've noticed this too and agree. I'm not a huge fan of `export styled(Component)` because then the styled component is anonymous I think. Maybe instead something like:

```js
class UnstyledSwitch extends React.Component {}

const Switch = styled(UnstyledSwitch);
export default Switch;
```

Reply:
yeah `export styled(Component)` is anonymous - I think we keep the short component name for the react component itself so that we can export it as a named export for tests.

```
class Switch extends React.Component {}

const StyledSwitch = styled(Switch);
export default StyledSwitch;
export {Switch};
```